adafruit_led_animation/color.py

```diff
@@ -41,8 +44,10 @@
 JADE = (0, 255, 40)
 AMBER = (255, 100, 0)
 
-RAINBOW = (RED, ORANGE, YELLOW, GREEN, BLUE, PURPLE)
+RGBW_RGB = (255, 255, 255, 0)
+RGBW_RGBW = (255, 255, 255, 255)
```

Comment:
What about RGBW_WHITE_RGB and RGBW_WHITE_RGBW and maybe also RGBW_WHITE_W

Reply:
You mean RGB_WHITE_RGB... etc?

Reply:
I wasn't sure `RGBW_RGB` is a clear colour name... This is what I was thinking.

```
RGBW_WHITE_RGB = (255, 255, 255, 0)
RGBW_WHITE_RGBW = (255, 255, 255, 255)
RGBW_WHITE_W = (0, 0, 0, 255)
```

Reply:
Isn't `RGBW_WHITE_*` redundant? The W in RGBW is the white pixel.

Reply:
I see what you're saying, since these are all various permutations of "WHITE" let's call the color WHITE and include which diodes are used to generate the white.  Yeah, I like that.

Reply:
Updated the PR with the proposed changes.